### ISSUE TITLE
Last verbal command spoken to an alexa as a string

### ISSUE THREAD

**Issue by MattL0** [Apollon77/ioBroker.alexa2#237]
Is it possible to get the last spoken command that we verbally said to an specific alexa . This would present itselft as a string into iobroker. One string per alexa echo device.

So if i say , alexa, turn of tv, to my living room echo. 

I will get a string’’ turn on tv’’ associated to a device in iobroker, and just that device.

Having, the real time string will permit a lot of automation.

Ex: alexa, turn of the tv in five hours. 

Iobroker ( with a set of triggers based of the string just received) will then turn off of the tv (let’s say the one in the living room) after a five hours timer .

**Comment by Apollon77** [comment]
This is available via "history" section already, but not via "per device" this is true. With a small JavaScript you could parse it

**Comment by MattL0** [comment]
Thank you ! Will play with this !

I am new to iobroker. I want to transfer all of this via node-red /mqtt to HomeSeer. 


I am at the point to try to get the cookie functioning with a us account.

**Comment by MattL0** [comment]
I ended up to change my amazon.de to amazon.com in the config files of the node_module itlseft.... works great.


was able to parse everything with my seven Alexa !
I can finally uninstall Openhab ( It had this function with the echocontrol binding).

I parse everything via nodered. Could be better but i am not a programmer.

**Comment by Apollon77** [comment]
A change on code level should not be needed, there are configuration values in Admin UI of the adapter available!

**Comment by MattL0** [comment]
Thanks, will look into it

**Comment by MattL0** [comment]
With IObroker. Installed on an Odroid xu4. I was able to remove my openhab install on my mains server ( echocontrolbinding) , i was able to remove home assistant fro my main server ( emby; Emby is really esier to play with on IObroker !!), and I was able to remove Nodered from my mains Server, since IObroker runs it as a service ( which is nice for monitoring ).


I am really happy with IoBroker. My install is so much simpler now !

I think you are the creator of IObroker? if so, thank a lot for this platform :)

Matt
